### PR TITLE
composer: fill ext-* dependencies

### PR DESCRIPTION
**Pull request by glensc**
based on https://github.com/pld-linux/ZendFramework/blob/auto/th/ZendFramework-1.12.20-1/ZendFramework.spec

which itself is based on:
- https://github.com/pld-linux/ZendFramework/commit/a61792aa4de46796d08413c9e5767a368c482ad9
- https://framework.zend.com/manual/1.12/en/requirements.introduction.html

replaces: https://github.com/zf1s/zend-validate/pull/3

cc @falkenhawk

### CODE REVIEW

File: packages/zend-service-yahoo/composer.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
         }
     },
     "suggest": {
+        "ext-dom": "Used in special situations or with special adapters"
         "zf1s/zend-validate": "Used in special situations or with special adapters"
```
Comment (falkenhawk) [inline packages/zend-service-yahoo/composer.json]: there is a json syntax error here (comma missing)

**Comment by glensc** [comment]
rechecked all packages with composer validate:

```
$ for a in packages/*/composer.json; do (cd $(dirname "$a") && pwd && composer.phar validate); done
/Users/glen/scm/tmp/zf1/packages/zend-acl
./composer.json is valid
/Users/glen/scm/tmp/zf1/packages/zend-amf
./composer.json is valid
...
```

**Comment by glensc** [comment]
here's some simpler version, using `$COMPOSER`:

```
➔ for COMPOSER in packages/*/composer.json; do COMPOSER=$COMPOSER composer.phar validate; done
packages/zend-acl/composer.json is valid
packages/zend-amf/composer.json is valid
packages/zend-application/composer.json is valid
packages/zend-auth/composer.json is valid
packages/zend-barcode/composer.json is valid
packages/zend-cache/composer.json is valid
packages/zend-captcha/composer.json is valid
^C
```

doc: https://getcomposer.org/doc/03-cli.md#composer

**Comment by glensc** [comment]
https://github.com/zf1s/zf1/pull/7

**Comment by glensc** [comment]
this is awesome, 

```
[Mon Jul 08 15:42:23.108338 2019] [:error] [pid 11] [client a.b.c.d:0] PHP Fatal error:  Call to undefined function ctype_alnum() in /app/vendor/zf1s/zend-uri/library/Zend/Uri.php on line 109
```

I was using 1.12.21, update to 1.13.0 will fix this ;)

**Pull request merged**